components/LoginForm.vue

```diff
@@ -73,27 +75,40 @@ export default Vue.extend({
     };
   },
   methods: {
+    ...mapActions('user', ['setUser', 'setUser']),
```

Comment:
```suggestion
    ...mapActions('user', ['setUser', 'clearUser']),
```
I might be wrong though